include/net/nrf_cloud_cell_pos.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_CELL_POS_H_
+#define NRF_CLOUD_CELL_POS_H_
+
+/** @file nrf_cloud_cell_pos.h
+ * @brief Module to provide nRF Cloud cellular positioning support to nRF9160 SiP.
+ */
+
+#include <zephyr.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup nrf_cloud_cell_pos nRF Cloud cellular positioning
+ * @{
+ */
+
+/** @brief Cell-based location request type */
+enum cell_based_location_type {
+	CELL_LOC_TYPE_INVALID = -1,
```

Comment:
Is there a need for CELL_LOC_TYPE_INVALID or could it be removed?

Reply:
Removed